cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/AwsManagementCluster.scss

```diff
@@ -0,0 +1,39 @@
+.aws-management-container {
+    position: relative;
+    .mgmt-cluster-admins-container {
```

Comment:
This is an example where we should probably try to use the css grid to set a container width. You'll want to use the cds-layout 'cols' attribute to set the width.